MidiKeyboard/build.gradle

```diff
@@ -4,12 +4,12 @@ def versionMajor = 1
 def versionMinor = 5
 
 android {
-    compileSdkVersion 33
+    compileSdkVersion 36
 
     defaultConfig {
         applicationId "com.mobileer.midikeyboard"
         minSdkVersion 23
-        targetSdkVersion 33
+        targetSdkVersion 36
         versionCode 6
```

Comment:
versionCode and versionName should be updated for all apps being changed and uploaded.